news/6790.bugfix.1.rst

```diff
@@ -0,0 +1,4 @@
+The :func:`~PyInstaller.utils.hooks.get_package_path` function now
```

Comment:
The problem is that `get_package_path` is used by contrib hooks, and could be used by other 3rd party hooks. So if we rework this function (so that it returns only the list of the paths - and leave the basedir magic to caller), we will probably need to mark the change as breaking.

The missing reference might be due to misspelled name (`get_package_path` instead of `get_package_paths`).

Reply:
Grrr, [it is as well](https://pyinstaller--6790.org.readthedocs.build/en/6790/hooks.html#PyInstaller.utils.hooks.get_package_paths).